tests/unit/http_/test_proxy.py

```diff
@@ -114,6 +114,32 @@ def test_get_with_different_base_url_plain_rule(self, router_server, httpserver:
         assert response.ok
         assert response.text == "ok/bar/"  # it's calling /bar/ because it's part of the root URL
 
+    def test_get_with_path_variable_does_not_match_empty_path(
```

Comment:
What's this test for?
- Is it to showcase an issue with the router? If so, it should be in a PR fixing the issue.
- Is it to test some behavior that is expected from the router? If so, it should not contain any comments like "as showcased... we currently solve..."

Reply:
second, it is about the fact that registering the route "<domain>/<path_path>". The confusion probably stems from this being a limitation, so neither bug nor feature. Should I remove the test alltogether?